### ISSUE TITLE
[iOS] CollectionView's filter bar difficult to access with VoiceOver

### ISSUE THREAD

**Issue by samglt** [jellyfin/Swiftfin#1734]
The filter bar at the top of a collection can't be selected by tapping and is in a strange position in the voiceover heirarchy. The filter buttons are located with the letter bar in between the bottom of the list of items and the bottom tab bar, when you'd expect them to be at the top in or near the navigation bar.

The similar filter bar at the top of the search tab doesn't have this issue for some reason

**Comment by LePips** [comment]
Closed by #2058
